src/module.c

```diff
@@ -7972,6 +7998,13 @@ int RM_GetServerVersion() {
     return REDIS_VERSION_NUM;
 }
 
+/**
+ * Return the RedisModuleTypeMethods version.
```

Comment:
how about something in these lines:
```suggestion
 * Return the current redis-server runtime value of REDISMODULE_TYPE_METHOD_VERSION.
 * You can use that when calling RM_CreateDataType to know which fields of
 * RedisModuleTypeMethods are gonna be supported and which will be ignored.
```

Reply:
Yes, Let me add.